overlays/dprint.nix

```diff
@@ -0,0 +1,61 @@
+{ dprint
+, jq
+, fetchurl
+, writeText
+, runCommand
+, makeWrapper
+
+, dprintJson
+}:
+
+let
+  dprintConfig = builtins.fromJSON (builtins.readFile dprintJson);
+
+  hashes = {
+    "https://plugins.dprint.dev/typescript-0.85.0.wasm" = "sha256-FMMoXkWL7oTZYCbh108dJtziUKgvvGH6FwK/cglUGLo=";
+  };
+
+  prefetchPlugin = url: fetchurl {
+    inherit url;
+    sha256 = hashes."${url}" or (throw "No known hash for dprint plugin '${url}', please update release.nix");
+  };
+
+  plugins = map prefetchPlugin dprintConfig.plugins;
+
+  simplifiedConfig = builtins.toJSON {
+    inherit plugins;
+  };
```

Comment:
For building the plugins, we only really need the `plugins` setting and nothing else. This also avoids unnecessary rebuilds when changing settings.